rascal-lsp/src/main/rascal/util/LanguageServer.rsc

```diff
@@ -214,13 +287,105 @@ data LanguageService
     | lenses(LensDetector detector)
     | inlayHinter(InlayHinter hinter)
     | executor(CommandExecutor executor)
-    | documenter(Documenter documenter)
-    | definer(Definer definer)
-    | referrer(Referrer reference)
-    | implementer(Implementer implementer)
+    | documenter(FocusDocumenter documenter)
+    | definer(FocusDefiner definer)
+    | referrer(FocusReferrer reference)
+    | implementer(FocusImplementer implementer)
     | actions(CodeActionContributor actions)
     ;
 
+@deprecated{This is a backward compatibility layer for the pre-existing ((Documenter)) alias.}
+@deprecated{
```

Comment:
are 2 deprecated tags allowed?

Reply:
Yes but the tutor will only see the first. Thx!